tests/linux/2024.vncjew/__min__c.simple

```diff
@@ -1,13 +1,71 @@
-# linux/2024.vncjew/__min__c: critical
-anti-static/elf/entropy: high
-anti-static/elf/header: high
-anti-static/elf/multiple: medium
-anti-static/packer/upx: high
-c2/addr/ip: high
+# linux/2024.vncjew/__min__c ∴ /__min__c: critical
+c2/addr/ip: medium
```

Comment:
any idea what change lowered it from high to medium?

Reply:
Looks like we went from:
```
bin_hardcoded_ip [HIGH] (1 string):
- 2.5.4.3
```
to
```
ip_port_mention [MED] (5 strings):
- IP
- lIp
- lookupPort
- oIp
- parsePort
```

Reply:
This should be a non-issue since we're scanning both files as part of [`f849ff6` (#731)](https://github.com/chainguard-dev/malcontent/pull/731/commits/f849ff659f7acd25b7a2f71e2b643ea13b027088).